app/helpers/sprangular/application_helper.rb

```diff
@@ -26,16 +26,23 @@ def js_environment
         config: {
           site_name: store.seo_title || store.name,
           logo:      asset_path(config.logo),
+          locale:    I18n.locale,
```

Comment:
Let's also add `supported_locales: [....]`, and we can use that to data in the ng-repeat for the language switcher